test.py

```diff
@@ -195,9 +227,9 @@ def timeSeconds(f):
   return html.escape("%.2f" % f)
 
 if not docker:
-  omc.sendExpression('setModelicaPath("%s")' % librariespath)
-omc_exe=os.path.join(omhome,"bin","omc")
-dygraphs=os.path.join(ompython_omhome or omhome,"share","doc","omc","testmodels","dygraph-combined.js")
+  omc.sendExpression('setModelicaPath("%s")' % librariespath.replace('\\','/'))
```

Comment:
`os.path.normpath` should also convert (back)slashes as appropriate for the platform?

Reply:
probably yes, I'll check

Reply:
The issue here is that OMPython needs double escaped `\\`, so on Windows it would be `\\\\`....
Ideally OMPython could handle this itself like OMJulia.jl does (https://openmodelica.github.io/OMJulia.jl/dev/api/).

omc does understand Linux style path on Windows, so this is the easy solution.

Reply:
> omc does understand Linux style path on Windows, so this is the easy solution.

That sounds like a straightforward solution. Could maybe also make all those `normpath` invocations redundant, if I understand correctly.